src/validation.h

```diff
@@ -103,7 +103,7 @@ void StopScriptCheckWorkerThreads();
 
 CAmount GetBlockSubsidy(int nHeight, const Consensus::Params& consensusParams);
 
-bool AbortNode(BlockValidationState& state, const std::string& strMessage, const bilingual_str& userMessage = bilingual_str{});
+bool FatalError(BlockValidationState& state, const std::string& strMessage, kernel::Notifications& notifications, const bilingual_str& userMessage = {});
```

Comment:
In commit "kernel: Add fatalError method to notifications" (0b88c307a8ed81705cf8e6fb6332fdf969eb0e2e)

Would maybe make `notifications` the first parameter. It seems awkward for it to placed between the debug message and the user message.